source/code-snippets/connection/srv.js

```diff
@@ -0,0 +1,23 @@
+const { MongoClient } = require("mongodb");
+
+// Connection URI
+const uri =
+  "mongodb+srv://sample-hostname:27017/?poolSize=20&useUnifiedTopology=true";
```

Comment:
suggestion (not required for lgtm):
perhaps this "sample-hostname" can be replaced with <sample-hostname> or <cluster-url> to be consistent the format we introduced in other code snippets. For example:

``const uri =
  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";`` is the format we've used for other snippets

Reply:
This snippet is slightly different in that I was directly referencing sample values in the explanation here: https://github.com/mongodb/docs-node/pull/43/files#diff-14c50d2a0fdcbcb3c3aaa64d68a8cc94R50
I thought it might be strange in this context:

> The next part of the connection string specifies the hostname or IP address and
port of your MongoDB instance. In the example above, we use ``sample-hostname``
as the hostname and ``27017`` as the port. Replace these values to point to
your MongoDB instance.

I could see something like the following if you think it's important for this to also be consistent:

> The next part of the connection string specifies the hostname or IP address and
port of your MongoDB instance. In the example above, we use ``sample-hostname``
as the hostname and ``27017`` as the port. Replace the `<cluster-url>` with your hostname and port, separated with a colon (e.g. `sample-hostname:27017`).

Reply:
I do like your suggested paragraph: 
`The next part of the connection string specifies the hostname or IP address and
port of your MongoDB instance. In the example above, we use sample-hostname
as the hostname and 27017 as the port. Replace the <cluster-url> with your hostname and port, separated with a colon (e.g. sample-hostname:27017).`  

It's much more clear than " Replace these values to point to
your MongoDB instance." 

Though I won't require the paragraph change for LGTM, I think it's more personal preference at this point - I get why the code snippet itself doesn't need to include `<cluster-url>`

Reply:
Replaced with a diagram that labels each part